src/index.js

```diff
@@ -237,17 +237,15 @@ class KadDHT extends EventEmitter {
    * Start listening to incoming connections.
    */
   start () {
-    if (this._running) {
-      return
-    }
-
     this._running = true
-    this.providers.start()
-    this._queryManager.start()
-    this.network.start()
 
-    // Start random walk, it will not run if it's disabled
-    this.randomWalk.start()
+    return Promise.all([
```

Comment:
Can you please add a breaking change message to the commit so that we do not forget it on merge?

Reply:
Done!